scripts/compare-coverage.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
I wonder if it is worth it to store somewhere on what commit was the comparison made. Both on main and current PR.

Reply:
I can see a value to output the commit SHA or maybe even the commit message on the comparison report.  Is that what you're thinking for the use case?

Reply:
My idea of the usecase is mainly sanity checking the thing I am comparing with is the latest / correct commit.